phpstan.neon.dist

```diff
@@ -0,0 +1,19 @@
+parameters:
+    level: 2
+
+    paths:
+        - src
+# Uncomment this, when phpstan extensions are allowed
+#        - tests
```

Comment:
I install all needed extensions it isn’t that working?

Reply:
The problem is, that we don't require phpunit as a dev-dependency. PHPStan can't find the `TestCase` class